meta-balena-common/recipes-support/development-features/development-features/development-features

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+DEVELOPMENT_FEATURES_FLAG_FILE="/var/volatile/development-features"
+
+. /usr/sbin/balena-config-vars
+
+if [ "${DEVELOPMENT_IMAGE}" = "true" ] && [ ! -f "${DEVELOPMENT_FEATURES_FLAG_FILE}" ]; then
+	echo "SSHD_OPTS='-f /etc/ssh/sshd_config_development'" > "${DEVELOPMENT_FEATURES_FLAG_FILE}"
```

Comment:
As per https://github.com/balena-os/meta-balena/pull/1816, I am considering an extra check here that would allow to configure development mode without password authentication.

Reply:
Using a config.json flag as mentioned?